CS3219-AY2324S1/ay2324s1-course-assessment-g36
# Add dockerization for user management app and questions service app

To test:
1. Have docker installed on your computer
2. In the `ay2324s1-course-assessment-g36` directory (aka outer most directory where the `docker-compose.yml` file is), run `docker compose up`. You should be able to see the app at `localhost:3000`. The questions and user services should be running as well.
